environments/.stackhpc/LEAFCLOUD.pkrvars.hcl

```diff
@@ -1,6 +1,7 @@
-flavor = "en1.xsmall"
+flavor = "ec1.medium"
```

Comment:
hmm, this seems like an expensive move, but I trust we need it for the root disk size?

Reply:
It ran out of memory on the 2GB en1.xsmall, so ec1.medium looked like the cheapest 4GB flavor